tools/ci/github-runners/v3/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt-get update && apt-get install --no-install-recommends --yes \
   libdbus-1-dev \
   libhdf5-dev \
   libluajit-5.1-dev \
+  libopusfile-dev \
```

Comment:
Please add this new dependency to the setup documentation as well.

Reply:
Done

Reply:
I think you forgot one. Or is opusfile installed on fedora by default?

Reply:
I cannot test, if that is the correct one.

Reply:
It is not installed by default, at least on a clean docker container (fedora38)
`opusfile-devel` is the Dev* package.

Reply:
So I assume this is the correct one?

Reply:
If libopusfile-dev was neede for Ubuntu then opusfile-devel should do the job. I'm building on my pc to confirm this :)

Reply:
Confirmed, `opusfile-devel` is required. Resolved.